features/support/env.rb

```diff
@@ -23,8 +23,13 @@
 
 Capybara.server = :webrick
 
+options = {
+  window_size: [1920, 1080],
+  timeout: 30
```

Comment:
Layout/HashAlignment: Align the keys and values of a hash literal if they span more than one line.